src/js/containers/LimitOrder/LimitOrder.js

```diff
@@ -67,6 +68,15 @@ export default class LimitOrder extends React.Component {
     clearInterval(this.invervalProcess)
   }
 
+  async getOrders() {
+    try {
+      const results = await limitOrderServices.getOrders();
```

Comment:
getOrders only call when user has login, should check user login

Reply:
Got it, thanks!